maven-plugin/src/main/java/org/openmrs/maven/plugins/packager/config/ValidateContentPackageMojo.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.maven.plugins.packager.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import com.vdurmont.semver4j.Semver;
+import com.vdurmont.semver4j.SemverException;
+
+/**
+ * The purpose of this Mojo is to validate content properties file - validates the properties are
+ * only in ranges. values like latest and next are not allowed.
+ */
+@Mojo(name = "validate-content-package")
+public class ValidateContentPackageMojo extends AbstractMojo {
+	
+	private static String RANGE_REGEX = ".*(\\^|~|>|>=|<|<=|\\|\\|| - ).*";
+	
+	// conf properties file
+	@Parameter(property = "sourceFile")
+	protected String sourceFile;
```

Comment:
It's helpful to provide Javadoc here that explains what the parameter is and what it is used for.